browser/metrics/metrics_reporting_util.cc

```diff
@@ -18,7 +18,7 @@ bool GetDefaultPrefValueForMetricsReporting() {
     case version_info::Channel::CANARY:
       return true;
     case version_info::Channel::UNKNOWN:
-      return false;
+      return true;
```

Comment:
TO-DO: This is for testing on Debug builds. Remove before merge.

Reply:
I think you don't need this change for testing after enabling crash report via settings.